src/Utils/GridHandler.js

```diff
@@ -20,11 +21,25 @@ class GridHandler {
         if (propogateState){
             this.gridUpdated();
         }
+
+        this.enableTileUpdates = true;
```

Comment:
I'd say `enableTileChanges`, as to me an `update` implies updates from any source human or computer, but `change` implies an action taken by a user

As in anything though, incredibly minor and just different viewpoints